<launch>
<node pkg="offline_planner" type="service_client" name="mission_service_client_node" output="screen" />
</launch>
